content/en/docs/kitex/Tutorials/advanced-feature/generic-call/basic_usage.md

```diff
@@ -713,6 +714,188 @@ func (g *GenericServiceImpl) GenericCall(ctx context.Context, method string, req
 
 ```
 
+### 5. Generic Call Mapping JSON to Protobufs  
```

Comment:
Please keep consistency with other titles.

`JSON/Protobuf Mapping Generic Call` suggested.

Reply:
A better approach would be to add a title level for the encoding, like:

```
- Thrift 
  - Binary Generic
  - ...
  - JSON Mapping Generic Call
- Protobuf
  - JSON Mapping Generic Call
```